src/components/event/experimental-api/ExperimentalAPIEventStore.ts

```diff
@@ -25,13 +27,35 @@ export class ExperimentalAPIEventStore {
 
 	@observable scrolledIndex: null | Number = null;
 
-	constructor() {
+	events: EventStoreDefaultStateType;
+
+	constructor(events: EventStoreDefaultStateType) {
+		this.events = events;
 		this.getRootIds();
+		this.loadEvent((events as any)?.selectedEventId);
```

Comment:
try doing it without using any.
first check if it's a string then it will be an id we need. if it's not a string check if its not a null